acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -625,17 +608,17 @@ def create_model_cos_zenith_binned(self,
                     f"{wobble} livetime per bin: {list(np.histogram(cos_zenith_observations, bins=cos_zenith_bin, weights=livetime_observations_and_wobble[i])[0].astype(int))}")
 
         # Create the dict for output of the function
-        dict_binned_model = {}
+        collection_binned_model = BackgroundCollectionZenith()
         for i in range(len(binned_model)):
-            dict_binned_model[np.rad2deg(np.arccos(bin_center[i]))] = binned_model[i]
+            collection_binned_model[np.rad2deg(np.arccos(bin_center[i]))] = binned_model[i]
```

Comment:
You could alternatively initiate it with a dict : 
```
        collection_binned_model = BackgroundCollectionZenith(
{np.rad2deg(np.arccos(bin_center[i]))] :  binned_model[i] for i in range(len(binned_model))}
)
```
Both are valid in my opinion.

Reply:
I would stick with the current code which is slightly easier to read in my opinion.